cactusbot/services/beam/constellation.py

```diff
@@ -0,0 +1,74 @@
+"""Interact with Beam Constellation."""
+
+import re
+import json
+
+from .. import WebSocket
```

Comment:
The code does not comply to PEP8.

*PEP8Bear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/cactusbot/services/beam/constellation.py
+++ b/cactusbot/services/beam/constellation.py
@@ -10,7 +10,7 @@
     """Interact with Beam Constellation."""
 
     URL = "wss://constellation.beam.pro"
-    
+
     RESPONSE_EXPR = re.compile(r'^(\d+)(.+)?$')
     INTERFACE_EXPR = re.compile(r'^([a-z]+):\d+:([a-z]+)')
 
```